lib/ansible/inventory/group.py

```diff
@@ -72,7 +72,7 @@ def deserialize(self, data):
         self.name = data.get('name')
         self.vars = data.get('vars', dict())
         self.depth = data.get('depth', 0)
-        self.hosts = data.get('hosts', {})
+        self.hosts = data.get('hosts', [])
```

Comment:
Could do self.hosts = data.get('hosts', tuple())  if this should actually be read-only.

Reply:
It isn't supposed to be read-only.

Reply:
no, its a list, this is where we ADD/REMOVE hosts to a group